format/spec.md

```diff
@@ -607,6 +611,8 @@ Notes:
 
 1. An alternative, *strict projection*, creates a partition predicate that will match a file if all of the rows in the file must match the scan predicate. These projections are used to calculate the residual predicates for each file in a scan.
 2. For example, if `file_a` has rows with `id` between 1 and 10 and a delete file contains rows with `id` between 1 and 4, a scan for `id = 9` may ignore the delete file because none of the deletes can match a row that will be selected.
+3. Floating point partition values are considered equal if there IEEE 754 floating-point “single format” bit layout
+are equal (the equivelant of calling `Float.floatToIntBits`` in Java).  The avro specification encodes all floating point values in this format.
```

Comment:
I don't understand the comment about Avro. What is the relevance here? All serialization formats should preserve the bits they are passed.

Reply:
Addressed typos I think.

@rdblue I clarified that NaNs are also normalized which is why I included this detail (since manifests are required to be in Avro the bits should be directly compatible.  I can drop that if you don't think it is worth mentioning.)